pom.xml

```diff
@@ -155,6 +155,10 @@
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>apache-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>s3</artifactId>
```

Comment:
scope this for testing if it is only used in unit tests

Reply:
```
<scope>test</scope>
```

Reply:
Sure.